dashboard/src/components/Config/PkgRepoList/PkgRepoForm.tsx

```diff
@@ -686,143 +732,210 @@ export function PkgRepoForm(props: IPkgRepoFormProps) {
-                              RepositoryStorageTypes.PACKAGE_REPOSITORY_STORAGE_CARVEL_IMAGE
-                            }
-                            onChange={handleTypeRadioButtonChange}
-                            required={plugin?.name === PluginNames.PACKAGES_KAPP}
-                          />
-                        </CdsRadio>
-                        <CdsRadio>
-                          <label htmlFor="kubeapps-repo-type-http">HTTP</label>
-                          <input
-                            id="kubeapps-repo-type-http"
-                            type="radio"
-                            name="type"
-                            disabled={!!repo?.type}
-                            value={
-                              RepositoryStorageTypes.PACKAGE_REPOSITORY_STORAGE_CARVEL_HTTP || ""
-                            }
-                            checked={
-                              type === RepositoryStorageTypes.PACKAGE_REPOSITORY_STORAGE_CARVEL_HTTP
-                            }
-                            onChange={handleTypeRadioButtonChange}
-                            required={plugin?.name === PluginNames.PACKAGES_KAPP}
-                          />
-                        </CdsRadio>
-                        <CdsRadio>
-                          <label htmlFor="kubeapps-repo-type-git">Git</label>
-                          <input
-                            id="kubeapps-repo-type-git"
-                            type="radio"
-                            name="type"
-                            disabled={!!repo?.type}
-                            value={
-                              RepositoryStorageTypes.PACKAGE_REPOSITORY_STORAGE_CARVEL_GIT || ""
-                            }
-                            checked={
-                              type === RepositoryStorageTypes.PACKAGE_REPOSITORY_STORAGE_CARVEL_GIT
-                            }
-                            onChange={handleTypeRadioButtonChange}
-                            required={plugin?.name === PluginNames.PACKAGES_KAPP}
-                          />
-                        </CdsRadio>
-                      </>
-                    )}
-                  </CdsRadioGroup>
+                  <>
+                    <CdsRadioGroup layout="vertical">
+                      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                      <label>Scope:</label>
+                      <CdsControlMessage>
+                        If the repository is global, its packages will be availabe in every
+                        namespace of the cluster. If it is namespaced-scoped, its packages will be
+                        available only in the namespace where the repository has been installed.
+                      </CdsControlMessage>
+
+                      {repo.namespaceScoped ||
+                        (isGlobalNamespace(namespace, plugin?.name, currentNsConfig) && (
```

Comment:
Using either the field from the repo object (editing the existing repo) or calculating if the ns is global or not.